qan-api2/exporters/slow_log.go

```diff
@@ -35,6 +35,7 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 
+	slowlogAgent "github.com/percona/pmm/agent/agents/mysql/slowlog"
```

Comment:
qan shouldn't import internal packages of pmm-agent. Please extract that function to utils.

Reply:
seems like we can drop whole this file without any harm. Please check if we use it anywhere